public/event-emitters/search-admin-event.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * When the user selects and administrative region in a country (polygon on the map).
+ */
+
+var _ = require('lodash');
+var P = require('pjs').P;
+
+var SearchAdminEvent = P({
+  key: 'SearchAdminEvent',
+  init: function(searched_admin_codes) {
+    if (!_.isArray(searched_admin_codes)) {
+      console.error("Bad argument to SearchAdminEvent", searched_admin_codes);
+    }
```

Comment:
We don't do checking for any of the other events that take arguments - is there a reason this is different?